plugins/modules/ipadnsrecord.py

```diff
@@ -641,7 +737,7 @@
   loc_lat_dir:
     description: LOC Direction Latitude
     required: false
-    choices: ["N", "S"]
+    type: str
```

Comment:
Shouldn't we keep the choices "N" and "S", as only them should be used (RFC 1876)?